frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,92 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+
+
+        @regression @to-do
+        Scenario: Setting Developer preference for perspective: UP-01-TC01
+            Given user in at admin perspective
+             When user clicks on user dropdown on masthead
+              And user clicks User Preference option
+              And user clicks on Perspective dropdown on User Preferences page
+              And user selects "Developer" option
```

Comment:
Same here, I believe its easier to implement this gherkin sentences later if the dropdown and selected value/option are in one action:

```suggestion
              And user changes user preference "Perspective" dropdown to "Developer"
```

Reply:
Updated